### DISCUSSION TITLE
[@types/styletron-react] Should add support for theming with typescript

### DISCUSSION THREAD

**Discussion by joseDaKing** [DefinitelyTyped/DefinitelyTyped#56240] [Issues with a @types package]
As of right now @types/styletron-react does not make it easy to use themes with typescript. I think the package should support theming like in the @types/styled-components.

Here is an example: https://blog.agney.dev/styled-components-&-typescript/

[a comment by an automated account is not shown]

**Comment by erictaylor** [comment]
Feel free to submit a PR.
